independent-projects/core-plugins/tika-plugin/bin/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is necessary as part of the PR? `independent-project` folder

Reply:
Removed!